model/src/main/java/org/cbioportal/model/ClinicalAttribute.java

```diff
@@ -0,0 +1,61 @@
+package org.cbioportal.model;
+
+import java.io.Serializable;
+
+public class ClinicalAttribute implements Serializable {
```

Comment:
I need to read up on lombok (first time I have heard about this) .. but in general I agree that we should be slow to add more frameworks that are not widely known to the development team. If we want to adopt a framework we should also not apply it in just one place, but across the codebase .. to keep the code approach consistent. (some frameworks might only apply to one area of course)